snuba/settings/__init__.py

```diff
@@ -239,9 +239,8 @@
 # Maximum jitter to add to the scheduling of threads of an optimize job
 OPTIMIZE_PARALLEL_MAX_JITTER_MINUTES = 30
 
-# File path for storage configs
-# TODO: This is hardcoded, the config files should be discovered automatically
-STORAGE_CONFIG_FILES_PATH = "./snuba/datasets/configuration/generic_metrics/storages"
+# File path glob for storage configs
+STORAGE_CONFIG_FILES_GLOB = "./snuba/datasets/configuration/**/storages/*.yaml"
```

Comment:
I don't know the idiom in python but I think you want this to be something like:
 `f"{__application_root__}/snuba/datasets/configuration/**/storages/*.yaml"` to avoid that issue Volo ran into earlier of only being able to run commands from application root (also deploys might start things from a different directory)